### PR TITLE
APP-94: configuration-java

### PR DESCRIPTION
**Pull request by hunch7**
APP-94: minor wording fix.

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

**Comment by Oldnickus** [comment]
@kaa-leeroy  add to whitelist

File: configurationdemo/resources/projects.xml [hunk 1]
```diff
@@ -26,24 +26,106 @@
...
-Now it's time to test things out: navigate to the Kaa server administrative interface by clicking the "Administrative console" link at the top of the Sandbox window. Log in as a tenant developer (devuser / devuser123), and navigate to the "Configuration demo" application, then select "Endpoint groups". From here you can manage endpoint groups and update endpoint configuration. Make sure you have tried updating configuration for endpoint group "All" and received it on your device. Do not forget to activate the updated configuration — otherwise the endpoint will not be able to receive it.<br/><br/>
-
-Please note that all the features of Administration UI are also supported by REST API.
-
-<h2>What's next?</h2>
-
-For more background on the Kaa Configuration subsystem please refer to the corresponding programming guide and design reference sections..
+<head>
+	<base href="http://kaaproject.github.io/kaa/docs/" target="_blank">
+</head>
```
Comment (krupet) [inline configurationdemo/resources/projects.xml]: Remove _head_ and _body_ tags.

File: configurationdemo/resources/projects.xml [hunk 2]
```diff
@@ -26,24 +26,106 @@
...
+
+<body>
+	<p>
+	    This sample application uses the <b>Configuration Extension</b> based on the <a href="./Glossary/#sdk-type">Kaa Java SDK</a>.
+		This extension enables <a href="./Glossary/#kaa-client">Kaa client</a> to update the <a href="./Glossary/#endpoint-ep">endpoint</a> configuration by receiving the data from <a href="./Glossary/#kaa-server">Kaa server</a>.
```
Comment (krupet) [inline configurationdemo/resources/projects.xml]: Replace shorted links with full ones.

**Comment by krupet** [comment]
LGTM

**Comment by vadimol** [comment]
Looks good for me

**Pull request merged**